packages/bbui/src/Popover/Popover.svelte

```diff
@@ -16,6 +16,9 @@
   export let direction = "bottom"
   export let showTip = false
 
+  let clazz
+  export { clazz as class }
```

Comment:
This is to support passing down a class from a parent component, is there a better way to do this? Also I had to mark the passed down class as `global` to get this to work, is that correct?

Reply:
Is this definitely needed? I can see that the reason you're doing this is to apply custom styles to the popover to effectively hardcode it to use a dark background, but can we not just let it respect themes as everything else does?